libyaml_q_declaration.md

```diff
@@ -22,46 +20,50 @@ It is also used in the [Go-yaml project](https://github.com/go-yaml/yaml), the p
 ROS 2 uses this library to load parameters from a file when the arguments are properly used. There are no official tests or code coverage information about this library, but in this case we are only interested in the portion of public API that reads YAML files and we will cover these
 tests and coverage in the vendored package. Including specific tests for the `libyaml` version imported by `libyaml_vendor` for the tier 1 platforms listed in [REP-2000](https://www.ros.org/reps/rep-2000.html#support-tiers).
 
-Considering the previously mentioned reasons, we consider this library to be robust and reliable and at Quality Level 1.
+Considering the previously mentioned reasons, we consider this library to be robust and reliable and at Quality Level 3.
```

Comment:
Should we say "robust and reliable" at quality level 3?

Reply:
It's a good question.

On the one hand, this is a pretty widely used and packaged library.  And we are depending on it in our core.  So we have to assume it is of reasonably high quality.

On the other hand, it doesn't fit our quality level definitions, and it seems like there have been [no commits](https://github.com/yaml/libyaml/commits/master) to it in a year.

What if I just change this to be "reliable"?  Thoughts?

Reply:
"reliable" sounds good to me.